README.md

```diff
@@ -1,4 +1,63 @@
-# Python Boilerplate For GitHub Tasks
+# Superheroes
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
 before starting.
+
+In this task, we will create superheroes using classes and inheritance.
+
+## Superhero Class
+The Superhero class represents a basic superhero. It has the following characteristics:
+
+### Attributes:
+- `name`: the name of the superhero;
+- `power_level`: the power level of the superhero.
+### Methods
+- `use_power()`: prints a message indicating that the superhero is using their power;
+- `calculate_power()`: calculates the power of the superhero based on their power level. Returns the calculated power multiplied by 2.
+
+Example:
+```python
+super_man = Superhero("Superman", 25)
+super_man.use_power()  # "Superman is using their power."
+print(super_man.calculate_power())  # "50"
+```
+
+## FlyingSuperhero Class
+The FlyingSuperhero class is a subclass of Superhero and represents a superhero with the ability to fly. 
+It has the following additional characteristics:
+
+### Attributes
+- `flight_speed_multiplier`: a multiplier that determines the superhero's flight speed.
+### Methods
+- `fly()`: prints a message indicating that the superhero is flying;
+- `calculate_power()`: overrides the calculate_power() method of the base class. 
```

Comment:
Is it planned here, not to specify any details about overriding `use_power()` method?